includes/rest/class-followers.php

```diff
@@ -97,6 +88,25 @@ function( $item ) {
 		return $response;
 	}
 
+	private static function get_followers( $request ) {
+		$user_id = $request->get_param( 'user_id' );
+		$order = $request->get_param( 'order' );
+		$per_page = $request->get_param( 'per_page' );
+		$page = $request->get_param( 'page' );
+		$offset = ( $page - 1 ) * $per_page;
+		$query = FollowerCollection::get_followers_query( $user_id, $per_page, $offset, array( 'order' => ucwords( $order ) ) );
```

Comment:
why not using `get_followers` if you make the transformation to a `Follower` anyways?

Reply:
I'm saving a separate call to `Followers::count_followers` by doing it this way, and guaranteeing that the identical query is the basis for the count.

Reply:
🤔 hmmm... good point...

Reply:
🤔 hmmm... good point...

Reply:
We could also possibly make some kind of internal API like

```php
$query = new WP_Query( $args );
$followers = transform_posts_to_followers( $query->posts );
$total = $query->found_posts;
return compact( 'followers', 'total' );
```

to just parse out what we consistently need from the queries